src/schema/schema_stat.c

```diff
@@ -159,8 +159,7 @@ __wt_curstat_table_init(
     if (table->is_simple) {
         WT_ERR(__wt_buf_fmt(session, buf, "statistics:%s", table->cgroups[0]->name));
         ret = __wt_curstat_init(session, buf->data, NULL, cfg, cst);
-        __wt_scr_free(session, &buf);
-        return (ret);
+        goto err;
```

Comment:
I would prefer having `goto done` rather than `goto err` as this path should not be interpreted as an error by any reader. A comment could be enough too instead of adding another label.

Reply:
I will add a comment here.

Reply:
Ah, it is better to have multiple labels, in this case, "done" or "skip" along with "err". An example will be:
https://github.com/wiredtiger/wiredtiger/blob/develop/src/block/block_ext.c#L1420

Reply:
Thanks, I'll follow this